syz-fuzzer/proc.go

```diff
@@ -274,6 +276,24 @@ func (proc *Proc) enqueueCallTriage(p *prog.Prog, flags ProgTypes, callIndex int
 	})
 }
 
+func (proc *Proc) execute(execOpts *ipc.ExecOpts, p *prog.Prog, flags ProgTypes, stat Stat) *ipc.ProgInfo {
```

Comment:
I would call this executeAndCollide and remove the return value, and keep executeNoCollide as just execute.
When I see just "execute" in the caller code, I assume it just executes the program once, but it does more complex logic.